<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Camera</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>500</y>
      <width>261</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Photo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Thema: Light</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>540</y>
      <width>161</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Video</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Location</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>540</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Camera: Local</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>548</x>
      <y>500</y>
      <width>71</width>
      <height>35</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>.png</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>.jpg</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>.jpeg</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>540</y>
      <width>71</width>
      <height>35</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>.mp4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>.avi</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>.mov</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>540</y>
      <width>101</width>
      <height>35</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>default FPS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>60 FPS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>30 FPS</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
